### ISSUE TITLE
🌿 Option to Hide Embeded Note Titles

### ISSUE THREAD

**Issue by ThisUserNeedsHelp** [jparkerweb/pixel-banner#81]
When one document is "embedded" (using the ![[document]] method) into another document, the pixel banner of the embedded document is misplaced to a position below the start of the text of the embedded document.

**Comment by jparkerweb** [comment]
@ThisUserNeedsHelp 
Thank you for the suggestion.

Embedded note titles always display above the note `content`, but I understand your frustration so I have added an option to the Settings page to hide embedded note titles (off by default).  Please update the plugin in Obsidian and take a look :)

Merry Christmas 🎅

---

🚩 Pixel Banner Release

⇢ Added ✨ New setting to hide embedded note titles

https://github.com/jparkerweb/pixel-banner/releases/tag/2.14.0

![Image](https://github.com/user-attachments/assets/5b1f68da-b319-452c-a143-e6bf13e68871)

**Comment by ThisUserNeedsHelp** [comment]
There appears to have been a misunderstanding.

I did not write anything about the Note Titles.

I wrote about the Pixel Banner. The Graphic Image.

Here is what I wrote:
When one document is "embedded" (using the ![[document]] method) into
another document, the pixel banner of the embedded document is misplaced to
a position below the start of the text of the embedded document.

There was no problem with the Note Titles.
